tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -42,6 +43,7 @@ sub reboot_and_wait_up {
     else {
         #leave ssh console and switch to sol console
         switch_from_ssh_to_sol_console(reset_console_flag => 'off');
+        send_key 'ret' if is_opensuse && !check_screen('text-logged-in-root');
```

Comment:
What does this line do?

Reply:
sometimes 'login:' has not prompted until 'Enter' is typed in sol console on my DELL SUTs, or black screen ~

Reply:
I would rather use `backspace` and I would not do that openSUSE specific.

Reply:
I use 'backspace' to have a try. If it works in SLE either, is_opensuse will be removed of course. Thanks.

Reply:
just tried 'backspace' does not work, have to revert to use 'ret'. http://10.67.129.51/tests/2510#step/reboot_and_wait_up_normal/6

but is_opensuse can be removed.